src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -29,13 +29,18 @@
 
 <H2>Cross Domain Script Inclusion</H2>
 Validates whether or not scripts are included from domains other than the domain hosting the content. 
-By looking at the "src" attributes of "script" tags in the response. 
-<ul><li>Allowed Cross-Domain Hosts: 
-At HIGH threshold if a script URL falls within a context that also includes the URL of the base message no alerts will be raised.</li></ul> 
+By looking at the "src" attributes of "script" tags in the response.<br>
+Allowed Cross-Domain scripts:  
+<ul>
+<li>Any script with a non empty "integrity" attribute is ignored - the integrity value is not checked as should will be checked by the browser</li>
```

Comment:
The last part of the sentence does not sound right ("... as it will be checked..." instead?).